lib/tasks/rollover.rake

```diff
@@ -0,0 +1,11 @@
+namespace :rollover do
+  desc "Rollover providers, courses and locations to the next cycle"
+  task :providers, [:provider_code] => :environment do |_task, args|
```

Comment:
In your service, you're doing `args.to_a.map(&:upcase)` which means you can pass in an array of provider codes. This makes is more obvious what you can pass in: 
```suggestion
  task :providers, [:provider_codes] => :environment do |_task, args|
```